lib/druid/post_aggregation.rb

```diff
@@ -268,4 +270,37 @@ def initialize(attributes = {})
     end
   end
 
+  class PostAggregationThetaSketch < PostAggregation
+    attr_accessor :field
+
+    class FuncValidator < ActiveModel::EachValidator
+      TYPES = %w(UNION INTERSECTION NOT)
+      def validate_each(record, attribute, value)
+        if TYPES.include?(record.type)
+          record.errors.add(attribute, 'may not be blank') if value.blank?
+        else
+          record.errors.add(attribute, "is not supported by type=#{record.type}") if value
+        end
```

Comment:
Isn't it more readable?

```
if TYPES.include?(record.type) && value.blank?
...
elsif value
...
end
```

Reply:
That's right. Maybe `elsif value` ?

Reply:
It will be more lines. Do you think it's more readable if it use `if`, `elsif` and `else`?

Also, I have used that way of condition statement because it's used in the another validators.